File: src/main/java/org/bk/graphql/repository/book/BookRepository.java
package org.bk.graphql.repository.book;

import org.bk.graphql.entity.BookEntity;
import org.springframework.data.repository.CrudRepository;
import org.springframework.data.repository.PagingAndSortingRepository;
import org.springframework.stereotype.Repository;

import java.util.UUID;

@Repository
public interface BookRepository extends CrudRepository<BookEntity, UUID>, PagingAndSortingRepository<BookEntity, UUID>, BookCustomRepository {
}

File: src/main/kotlin/org/bk/graphql/repository/BookRepository.kt
package org.bk.graphql.repository

import org.bk.graphql.entity.BookEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.data.repository.PagingAndSortingRepository
import org.springframework.stereotype.Repository

@Repository
interface BookRepository: CrudRepository<BookEntity, String>, PagingAndSortingRepository<BookEntity, String>